PC/layout/support/options.py

```diff
@@ -53,7 +53,16 @@ def public(f):
     },
     "nuget": {
         "help": "nuget package",
-        "options": ["stable", "pip", "distutils", "dev", "props"],
+        "options": [
+            "dev",
+            "tools",
+            "pip",
+            "stable",
+            "distutils",
+            "venv",
+            "launchers",
```

Comment:
I suspect we don't actually want this one. The original option was "launcher" (no "s"), which means it isn't including this. 

IIRC, "launchers" is specific to the APPX package.

Reply:
Yeah, the ``include_launchers`` option is only referenced in one place and looks like this:

```python
if ns.include_launchers and ns.include_appxmanifest:
    if ns.include_pip:
        yield from in_build("python_uwp.exe", new_name="pip")
    if ns.include_idle:
        yield from in_build("pythonw_uwp.exe", new_name="idle")
```

So no need for it here

Reply:
Thanks, fixed.